manual/cypher/cypher-docs/src/docs/graphgists/intro/compose-statements.adoc

```diff
@@ -31,14 +46,23 @@ You use the `WITH` clause to combine the individual parts and declare which data
 `WITH` is very much like `RETURN` with the difference that it doesn't finish a query but prepares the input for the next part.
 You can use the same expressions, aggregations, ordering and pagination as in the `RETURN` clause.
 
-The only difference is that you _have to_ alias all columns as they would otherwise not be accessible with an identifier.
-Every column that you don't declare in your `WITH` clause is not available in subsequent query parts.
+The only difference is that you _must_ alias all columns as they would otherwise not be accessible.
+Only columns that you declare in your `WITH` clause is available in subsequent query parts.
+
+See below for an example where we collect the movies someone appeared in, and then filter out those which appear in more than one movie.
```

Comment:
Technically, we keep those. We filter out those which _don't_ appear in more than one movie.